examples/example_plugin/pyproject.toml

```diff
@@ -16,3 +16,33 @@ python = ">=3.8,<3.12"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.ruff]
+line-length = 120
+target-version = "py38"
+
+[tool.ruff.lint]
+select = [
+    "DJ",  # flake8-django
+    "E",  # pycodestyle
+    "F",  # pyflakes
+    "I",  # isort
+    "PIE",  # flake8-pie
+    "RUF",  # ruff itself
+    "S",  # flake8-bandit
+    "W",  # pycodestyle
+]
+ignore = [
+    "E501",  # pycodestyle: line-too-long
+    "PIE808",  # unnecessary-range-start
+    "RUF012",  # mutable-class-default - see https://github.com/astral-sh/ruff/issues/5243
+]
+
+[tool.ruff.lint.isort]
+combine-as-imports = true
+force-sort-within-sections = true
+order-by-type = false
+section-order = ["future", "standard-library", "third-party", "nautobot", "first-party", "local-folder"]
+
+[tool.ruff.lint.isort.sections]
+"nautobot" = ["nautobot"]
```

Comment:
No idea why GitHub is considering this invalid - as far as I can tell it's correct TOML?

Reply:
Github does not like a comment on the last line of the list within toml... seems like a github bug, will see if I can open an issue. 

![image](https://github.com/nautobot/nautobot/assets/9260483/960f7f73-7695-4930-af90-4231bf7ac163)

Reply:
Opened github support ticket 2514042 on it